Captricity/kombu-redis-priority
# Support new version of redis which changes zadd signature

For @mauesrog,

This updates this library to support newer versions of the redis client library.

Two fixes:
- Change the args to zadd
- Change `async` args to `asynchronous`

**Background**
This library is base on code pulled from the [kombu](https://github.com/celery/kombu/blob/master/kombu/transport/redis.py) library. We literally copied the code from that file and updated it to use sorted sets. To fix the issues here, I pulled this file from the 4.4.0 version of kombu and compared it to our version of the file. I then updated our version where needed.

**Tests**
https://travis-ci.org/Captricity/kombu-redis-priority

**Manual Testing**
I have a [cipher branch](https://github.com/Captricity/Cipher/pull/1942) that uses this branch (along with other celery/redis related changes). Check out that branch and try digitizing something through cipher.
